miner/worker.go

```diff
@@ -652,6 +652,9 @@ func (env *Work) commitTransactions(mux *event.TypeMux, transactions types.Trans
 	if len(coalescedLogs) > 0 {
```

Comment:
Could you change this to

```
go func(logs vm.Logs, tcount int) {
    if len(coalescedLogs) > 0 {
        mux.Post(....)
    }
    if tcount > 0 {
        mux.Post(...)
    }
}(coalescedLogs, env.tcount)
```

so this can be done from one goroutine rather than 2 separate ones.